server/src/domain/person/person.service.ts

```diff
@@ -57,6 +79,13 @@ export class PersonService {
       await this.jobRepository.queue({ name: JobName.SEARCH_INDEX_ASSET, data: { ids } });
     }
 
+    if (dto.isHidden !== undefined) {
+      person = await this.repository.update({ id, isHidden: dto.isHidden });
+      const assets = await this.repository.getAssets(authUser.id, id);
+      const ids = assets.map((asset) => asset.id);
+      await this.jobRepository.queue({ name: JobName.SEARCH_INDEX_ASSET, data: { ids } });
+    }
```

Comment:
Can you combine this if statement with the one above? It should look like this:
```typescript
    if (dto.name || dto.isHidden !== undefined) {
      person = await this.repository.update({ id, name: dto.name, isHidden: dto.isHidden });
      const assets = await this.repository.getAssets(authUser.id, id);
      const ids = assets.map((asset) => asset.id);
      await this.jobRepository.queue({ name: JobName.SEARCH_INDEX_ASSET, data: { ids } });
    }
```

If name is undefined or if isHidden is undefined, passing it to the update method won't do anything.